jest.config.js

```diff
@@ -0,0 +1,26 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
```

Comment:
Since we dropped react-scripts, we now have to maintain our own config.

This is greatly improved over our previous setup, where we were passing some configuration options via the package.json config, and others via the command line, because react-scripts doesn't allow overriding all of the options we need to override.